packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -679,6 +683,28 @@ class ThemeData extends Diagnosticable {
   /// Configures the hit test size of certain Material widgets.
   final MaterialTapTargetSize materialTapTargetSize;
 
+  /// Apply a semi-transparent white overlay on Material widgets when in dark themes.
+  ///
+  /// If [true] a semi-transparent white overlay will be applied to the surface
+  /// color of [Material] widgets when the surrounding theme's brightness is dark.
+  /// The transparency is based off of the [elevation] as per the Material Dark
+  /// theme specification.
+  ///
+  /// If [false] the surface color will be used unmodified.
+  ///
+  /// Defaults to [false].
+  ///
```

Comment:
Same comment as earlier about explaining the need for this (dark shadows, etc).

It looks like there are some opportunities for using `@template` and `@macro` here.